ansible/roles/test/files/ptftests/fg_ecmp_test.py

```diff
@@ -0,0 +1,409 @@
+# PTF test contains the test cases for fine grained ecmp, the scenarios of test are as follows:
```

Comment:
how do we make sure that this run only for given platform ?

Reply:
Added platform check, thanks!